packages/browsers/src/browsers/firefox.ts

```diff
@@ -88,3 +89,256 @@ export async function resolveBuildId(
     });
   });
 }
+
+export async function createProfile(
+  extraPrefs: {[x: string]: unknown},
```

Comment:
Let's use the `ProfileOptions` (I think we will need to put it in `types.ts`) type here either passing a single argument (destructing it) or with specific  key from said type.